Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+flake8 = "*"
+
+[requires]
+python_version = "*"
```

Comment:
Did you run the command `pipenv --python 3` to create the `Pipfile`?
Because usually when running it we get `python_version = "3.9"` (or whichever python 3 version you have in your environment).

Reply:
@avi-biton Yes I did, but in Challenge #1 `python_version = "3.9"` came out with an error in `Setup environment`, and after I changed to `python_version = "*"` worked and passed all the tests successfully.

Reply:
OK, thanks.
Lets leave it for now, we will fix it later if we will have issues